css/style.css

```diff
@@ -16,4 +21,135 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/*header*/
 
+ header {
+     background-color: white;
+     width: 100%;
+     min-height: 20px;
+     position: absolute;
+    }
+
+ header ul {
+     display: flex;
+     justify-content: flex-end;
+} 
+
+header li {
+    list-style: none;
+    padding-right: 20px;
+}
+
+ header a {
+     color: rgb(187, 180, 180);
+     text-decoration: none;
+     font-size: 20px;
+}   
+
+
+li:first-child {
+    margin-right: auto;
+}
+
+header a:hover {
+    color: rgb(46, 39, 39);
+}
+
+span {
+    color: rgb(43, 41, 41);
+}
+
+span:hover {
+    color: rgb(161, 153, 153);
+}
+
+header img {
+    width: 40px;
+    height: 40px;
+}
+
+/*main*/
+
+.section1 {
+    width: 60%;
+    padding: 20%; 
+    line-height: normal;
+    color: rgb(248, 246, 252);
+    background-image: url('../img/first-background.jpg');
+    background-repeat: no-repeat;
+    background-position: auto;
+    background-size: 100% 100%;
+    
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+h1 {
+    font-size: 50px;
+}
+.button {
```

Comment:
Try to be consistent with formatting, there should be a new line at the end of this css statement as you have done elsewhere in the file.